tests/functional/prometheous/test_prometheus_validations.py

```diff
@@ -974,3 +975,81 @@ def test_restart_prometheus_glusterfs_pod(self):
             # Try to fetch metric from prometheus pod
             self._fetch_metric_from_promtheus_pod(
                 metric='heketi_device_brick_count')
+
+    @pytest.mark.tier2
+    def test_metrics_workload_on_prometheus(self):
+        """Validate metrics workload on prometheus"""
+
+        # Skip test if the prometheus pods are not present
+        openshift_ops.switch_oc_project(
+            self._master, self._prometheus_project_name)
+        prometheus_pods = openshift_ops.oc_get_pods(
+            self._master, selector=self._prometheus_resources_selector)
+        if not prometheus_pods:
+            self.skipTest(
+                prometheus_pods, "Skipping test as prometheus"
+                " pod is not present")
+
+        self.prefix = "autotest-{}".format(utils.get_random_str())
+        if not self.registry_sc:
+            self.skipTest(
+                prometheus_pods, "Skipping test as registry "
+                " storage details are not provided")
+        self._registry_project = self.registry_sc.get(
+            'restsecretnamespace')
+
+        # Get one of the prometheus pod name and respective pvc name
+        prometheus_pod = list(prometheus_pods.keys())[0]
+        pvc_custom = ":.spec.volumes[*].persistentVolumeClaim.claimName"
+        pvc_name = openshift_ops.oc_get_custom_resource(
```

Comment:
use 
```
pvc_name = openshift_ops.oc_get_custom_resource(
              self._master, "pod", pvc_custom, prometheus_pod)[0]
```
And now check for assertTrue, Now you can ignore line number 1009

Reply:
You mean:
        pvc_name = openshift_ops.oc_get_custom_resource(
            self._master, "pod", pvc_custom, prometheus_pod)[0[
        self.assertTrue(
            pvc_name, "Failed to get PVC name for prometheus"
            " pod {}".format(prometheus_pod))
        self.verify_iscsi_sessions_and_multipath(
            pvc_name, prometheus_pod, rtype='pod',
            heketi_server_url=self._registry_heketi_server_url,
            is_registry_gluster=True)

Reply:
Right.

Reply:
Done